docs/components/buttongroup/usage.md

```diff
@@ -1,2 +1,3 @@
 This component is only supported in Warp Vue at the moment.
+
 For a React equivalent that serves a similar purpose check the Toggle<!-- [Toggle](/components/toggle/) --> with `radio-button` type.
```

Comment:
That's probably my bad as I was waiting for the toggle page to be deployed first. Do you think you could use the hyperlink here and test it works fine? 🙏